pages/searching/csv_export.rst

```diff
@@ -1,17 +1,41 @@
 Export results as CSV
 ^^^^^^^^^^^^^^^^^^^^^
-It is also possible to export the results of your search as a CSV document. To do so, click on the three dots on the right side of the search bar and select the *Export as CSV* option.
+It is possible to export the results of your search as a CSV document. To do so, click on the three dots on the right side of the search bar and select the *Export to CSV* option.
 
 .. image:: /images/searching/csv_export.png
    :align: center
 
-This will open a modal where you can choose the fields the CSV document should contain. You can also optionally specify a stream to filter the messages.
-Clicking on *Downlaod* will create the file based on the specified options and start the download.
+This will open a modal where you can choose the fields the CSV document should contain. The initial fields are based on the message table configured for the search page.
+It is also possible to define a limit. Because messages are loaded in chunks, all chunks up to the one where the limit is reached will be retrieved. Which means the total number of delivered messages can be higher than the defined limit.
 
-.. image:: /images/searching/csv_export_dialog.png
+Clicking on *Start Download* will create the file based on the specified options and start the download.
+
+.. image:: /images/searching/csv_export_dialog_v2.png
+   :align: center
+
+When the search page or saved search contains multiple message tables, you can decide which message table you want to adopt the settings from. In this case you will see the following select when opening the CSV export modal:
+
+.. image:: /images/searching/csv_export_widget_select.png
+   :align: center
+
+Additionally, you can select *Export to CSV* in a message table’s action menu, if you want to adopt its settings directly.
+
+.. image:: /images/searching/csv_export_message_table_action_menu.png
    :align: center
 
-**Hint**: Some Graylog inputs keep the original message in the ``full_message`` field. If you need to export the original message, you
-can do so by clicking on the *List all fields* link at the bottom of the sidebar, and then selecting the ``full_message`` field.
+Exporting search results on a dashboard
```

Comment:
How about "Exporting Message Tables on a Dashboard"? That would set expectations for what's exportable more precisely, I think.